jans-auth-server/server/src/main/java/io/jans/as/server/service/MDSUpdateTimer.java

```diff
@@ -0,0 +1,67 @@
+package io.jans.as.server.service;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Event;
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import io.jans.fido2.service.mds.TocService;
+import io.jans.service.cdi.async.Asynchronous ;
+import io.jans.service.cdi.event;.Scheduled;
+import io.jans.service.timer.event;
+import io.jans.service.timer.schedule.TimerSchedule;
+import org.slf4j.Logger;
+
+/**
+ * @author madhumitas
+ *
+ */
+@ApplicationScoped
+@Named
+public class MDS3UpdateTimer {
```

Comment:
It's not clear why timer with event is put in jans-auth-server. Shouldn't it be in fido2-server? fido2-server is separate war and can carry it.

Reply:
This was in error, corrected